plugin.idea/src/com/microsoft/alm/plugin/idea/common/setup/LinuxStartup.java

```diff
@@ -32,7 +32,7 @@ public static void startup() {
         try {
             final File script = new File(IdeaHelper.getResourcePath(LinuxStartup.class.getResource("/"), SCRIPT_NAME, LINUX_DIR));
             IdeaHelper.setExecutablePermissions(script);
-            createDesktopFile(script, new File(USER_HOME, VSOI_DESKTOP_FILE_PATH));
+            createDesktopFileAndUpdateDatabase(script, new File(USER_HOME, VSOI_DESKTOP_FILE_PATH));
```

Comment:
Thanks for making this change,  we also need it updated in the unit test [here](https://github.com/shalupov/vso-intellij/blob/051f36bdc8d02b8a120643a2c34ef1d37bb4c28e/plugin.idea/test/com/microsoft/alm/plugin/idea/common/setup/LinuxStartupTest.java#L23) so everything complies

Reply:
oops, fixed